shell/platform/windows/flutter_windows_view.h

```diff
@@ -206,6 +213,36 @@ class FlutterWindowsView : public WindowBindingHandlerDelegate {
 
   // Currently configured WindowBindingHandler for view.
   std::unique_ptr<flutter::WindowBindingHandler> binding_handler_;
+
+  // Resize events are synchronized using this mutex and the corresponding
+  // condition variable.
+  std::mutex resize_mutex_;
+  std::condition_variable resize_cv_;
+
+  // States a resize event can be in.
+  enum class ResizeState {
```

Comment:
Nit: let's move this to the top of the private block, rather than having a type declaration mixed in with the ivars.

Reply:
Done